src/style.css

```
@@ -0,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+.label {
+  display: block;
+}
+
+.form {
+  box-sizing: border-box;
+  width: 100%;
+  margin-bottom: 20px;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: 900;
+  color: rgba(61, 78, 97, 1);
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(Images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.FirstInput {
```

```suggestion
.firstInput {
```
класи прийнято писати з малої літери